features/step_definitions/projects_page_steps.rb

```diff
@@ -0,0 +1,163 @@
+# frozen_string_literal: true
+
+Then('I should see {string} public project card in {string} team page') do |project, team|
+  expect(page).to have_selector '.nav-name', text: team
+  expect(page).to have_selector '.panel-project a', text: project
+  expect(page).to have_selector '.panel-project span', text: 'All team members'
+end
+
+Then('I should see {string} private project card in {string} team page') do |project, team|
+  expect(page).to have_selector '.nav-name', text: team
+  expect(page).to have_selector '.panel-project a', text: project
+  expect(page).to have_selector '.panel-project span', text: 'Project members only'
+end
+
+Then('I should see {string} archived project card in {string} team page') do |project, team|
+  expect(page).to have_selector '.nav-name', text: team
+  expect(page).to have_selector '.panel-project .panel-title', text: project
+  expect(find('.projects-view-filter.active')).to have_content('ARCHIVED')
+end
+
+Then('I should see {string} error message of {string} modal window') do |message, modal|
+  modal_object = page.find('.modal-content', text: modal)
+  expect(modal_object).to have_selector '.help-block', text: message
+end
+
+Given('I had project {string} for team {string}') do |project, team|
+  FactoryBot.create(:project, name: project, team: Team.find_by(name: team))
+end
+
+Then('I click to down arrow of a {string} project card') do |project|
+  page.find('.panel-project', text: project).hover.find('.caret').click
+end
+
+Then('user {string} owner of project {string}') do |user, project|
+  FactoryBot.create(:user_project,
+                    role: 0,
+                    user: User.find_by(full_name: user),
+                    project: Project.find_by(name: project))
+end
+
+Then('user {string} normal user of project {string}') do |user, project|
+  FactoryBot.create(:user_project,
+                    role: 1,
+                    user: User.find_by(full_name: user),
+                    project: Project.find_by(name: project))
+end
+
+Given('I click to {string} of a Options modal window') do |link|
+  page.find('.panel-project .dropdown-menu', text: 'Options').find('a', text: link).click
+end
+
+Then('I click {string} icon on {string} project card') do |icon, project|
+  page.find('.panel-project', text: project).find(".fa-#{icon}").click
+end
+
+Then('I select user {string} in user dropdown of user manage modal for project {string}') do |user, project|
+  within('.modal-content', text: "Manage users for #{project}") do
```

Comment:
i18n

Reply:
Scenarios should be improved with common actions, then this file can be removed. No improvements for now.